### ISSUE TITLE
changing position/size of State-wise COVID-19 Cases India

### ISSUE THREAD

**Issue by xAdvitya** [anamritraj/livecovid.in-webapp#31]
In order to use the slider user needs to scroll down and then scroll up again to see the data even with browser on full screen,It would be more convenient if the whole map card is visible in the screen without scrolling .
![Screenshot (251)](https://user-images.githubusercontent.com/54709416/82134355-94b6d780-9814-11ea-8965-9fe2a14f3e1f.png)

**Comment by anamritraj** [comment]
Yeah this should be be an easy one. Can you please also let me know what browser/device is this?

**Comment by xAdvitya** [comment]
> Yeah this should be be an easy one. Can you please also let me know what browser/device is this?

yes,its vivaldi
